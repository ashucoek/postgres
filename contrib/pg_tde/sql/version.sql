CREATE EXTENSION pg_tde;
SELECT pg_tde_version();
DROP EXTENSION pg_tde;
