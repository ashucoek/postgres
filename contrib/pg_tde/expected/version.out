CREATE EXTENSION pg_tde;
SELECT pg_tde_version();
 pg_tde_version 
----------------
 pg_tde 1.0.0
(1 row)

DROP EXTENSION pg_tde;
